MapzenSDK/Marker.swift

```diff
@@ -256,7 +256,7 @@ public class PointMarker : GeometricMarker, GenericPointIconMarker {
   /**
    Initializes a marker with a given size.
 
-   - parameter size: The size the marker should be.
+   - parameter s: The size the marker should be.
```

Comment:
Would there be some nicer code style to adopt concerning this without shading the type or private class names? Not sure what's swifty around this kind of things but one letter parameters for public interface seem to be ambiguous.